src/main/java/org/example/C01_10_MultiplyOfIntegers.java

```diff
@@ -0,0 +1,10 @@
+package org.example;
+
+public class C01_10_MultiplyOfIntegers {
+    public static int multiply(int num1, int num2) {
+        int multiply = num1 * num2;
+        return multiply;
```

Comment:
Use correct English. Multiply is a verb. The result of multiplying is `multiplication` or `product`.